tests/models/registry.py

```diff
@@ -258,6 +258,7 @@ def check_available_online(
     "Zamba2ForCausalLM": _HfExamplesInfo("Zyphra/Zamba2-7B-instruct"),
     "MiMoForCausalLM": _HfExamplesInfo("XiaomiMiMo/MiMo-7B-RL",
                                         trust_remote_code=True),
+    "Dots1ForCausalLM": _HfExamplesInfo("rednote-hilab/dots.llm1.inst"),
```

Comment:
```suggestion
    "Dots1ForCausalLM": _HfExamplesInfo("rednote-hilab/dots.llm1.inst", min_transformers_version="4.53"),
```
Since latest `Transformers` hasn't been released, we need to add `min_transformers_version` to skip this model temporarily.